pom.xml

```diff
@@ -112,6 +139,32 @@
 
 	<build>
 		<plugins>
+			<plugin>
```

Comment:
@Frawless I do not think you should be adding this. The assembly is done on L198-215.

Reply:
@scholzj redundant assembly removed